app/views/admin/invitations/index.html.erb

```diff
@@ -1,123 +1,91 @@
-      <div class="flex flex-col m:hidden fs-s">
-        <%= form_with url: admin_invitations_path, method: :get, local: true, class: "py-3" do |f| %>
-          <div class="flex justify-content-between">
-            <%= render "admin/users/index/expand_search_button" %>
-            <%= paginate @invitations, theme: "admin", scope: @invitations, label: "Paginate invitations", context: "top-small" %>
-          </div>
-          <div id="search-users" class="hidden mt-3">
-            <%= render "admin/users/index/search_field", f: f, placeholder: "Search invited members...", aria_label: "Search invited members by name, username, or email", context: "responsive" %>
-          </div>
-        <% end %>
-        <%= link_to "Invite member", new_admin_invitation_path, class: "c-cta c-cta--branded ml-2 inline-block s:self-end" %>
-
-        <h2 class="member-data-heading fs-s py-2 mt-3 -mx-4 px-6 color-base-60">Invited members</h2>
-
-        <ul class="list-none mx-2">
-          <% @invitations.each do |user| %>
-             <li class="py-4">
-              <article class="flex justify-content-between items-start">
-                <div class="flex">
-                  <%= render "admin/users/index/member_image", user: user %>
-                  <div class="flex flex-col gap-2">
-                    <%= render "admin/users/index/member_data", user: user %>
-                    <%= user.email %>
-                    <dl>
-                      <dt class="color-base-60 fw-normal">Invited on</dt>
-                      <dd><%= user.created_at.strftime("%d %b, %Y") %></dd>
-                    </dl>
-                  </div>
-                </div>
-                <%= render "admin/users/index/invitation_actions_dropdown", user: user, context: "small" %>
-              </article>
-             </li>
-          <% end %>
-        </ul>
-      </div>
-      <!-- Small screen layout end -->
-
-      <!-- Large screen layout start -->
-      <div class="hidden m:flex flex-col m:flex-row justify-content-between p-3 xl:px-0">
-        <%= form_with url: admin_invitations_path, method: :get, local: true, class: "grow-1 mb-3 m:mb-0" do |f| %>
-          <% render "admin/users/index/search_field", f: f, placeholder: "Search invited members...", aria_label: "Search invited members by name, username, or email", context: "large" %>
-        <% end %>
-        <div class="flex grow-1 justify-content-end">
-          <%= paginate @invitations, theme: "admin", scope: @invitations, label: "Paginate invitations", context: "top-large" %>
-          <%= link_to "Invite member", new_admin_invitation_path, class: "c-cta c-cta--branded ml-3" %>
+    <!-- Small screen layout start -->
+    <div class="flex flex-col m:hidden fs-s">
+      <%= form_with url: admin_invitations_path, method: :get, local: true, class: "py-3" do |f| %>
+        <div class="flex justify-content-between">
+          <%= render "admin/users/controls/expand_search_button" %>
```

Comment:
You'll notice that I move some of the controls files to be in a "controls" directory.  I've removed it from the "index" folder since its been used by invitations as well.